src/Serilog.Sinks.AzureDataExplorer.Tests/AzureDataExplorerSinkTest.cs

```diff
@@ -0,0 +1,169 @@
+using System.Reflection;
+using Kusto.Ingest;
+using Serilog.Events;
+using Serilog.Sinks.AzureDataExplorer.Sinks;
+
+namespace Serilog.Sinks.AzureDataExplorer.Tests
+{
+    public class AzureDataExplorerSinkTest
+    {
+        [Fact]
+        public void Test_constructor_should_throw_exception_if_options_is_null()
+        {
+            // Arrange, Act, Assert
+            var ex = Assert.Throws<ArgumentNullException>(() => new AzureDataExplorerSink(null));
+            Assert.Equal("options", ex.ParamName);
+        }
+
+        [Fact]
+        public void Test_constructor_should_throw_exception_if_database_name_is_null()
+        {
```

Comment:
See prior comments, we can possibly look at eliminating duplicate tests between Durable/Non-Durable sinks ? Whats your thought